osu.Game/Beatmaps/MetadataUtils.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System.Linq;
+using System.Text;
+
+namespace osu.Game.Beatmaps
+{
+    /// <summary>
+    /// Groups utility methods used to handle beatmap metadata.
+    /// </summary>
+    public static class MetadataUtils
+    {
+        /// <summary>
+        /// Returns <see langword="true"/> if the character <paramref name="c"/> can be used in <see cref="BeatmapMetadata.Artist"/> and <see cref="BeatmapMetadata.Title"/> fields.
+        /// Characters not matched by this method can be placed in <see cref="BeatmapMetadata.ArtistUnicode"/> and <see cref="BeatmapMetadata.TitleUnicode"/>.
+        /// </summary>
+        public static bool IsRomanised(char c) => c <= 0xFF;
```

Comment:
Stable actually has two different checks for diacritics in metadata. One, in the actual editor setup screen, only allows basic 7-bit ASCII, while one in AiMod allows extended ASCII. This is in line with the latter. Not sure if that's fine or not. I tested lightly with stable and it didn't seem to die on extended ASCII.

[There was an issue](https://github.com/ppy/osu/issues/6552) asking for diacritics support. This will allow *some* diacritics; however this will *not* accept all Latin diacritics (e.g. some Polish ones don't make it into this range, among *many* others). Latin script encompasses [1.3k characters](https://en.wikipedia.org/wiki/Latin_script_in_Unicode) and the line gets fuzzy and arbitrary wherever you place it.